### DISCUSSION TITLE
Improving Vivify's visibility (e.g. Google SEO)

### DISCUSSION THREAD

**Discussion by Tweekism** [jannis-baum/Vivify#121] [General]
Continuing on from https://github.com/jannis-baum/Vivify/issues/101#issuecomment-2248048497

Raising the discussion here about how to improve this projects standing on Google's search results. Figure this is the best place to talk about it until there are specific actions that can be raised as issues in the issue tracker.

Seriously, I only got here by trolling through ancient pull requests on iamcco's old plug-in repo looking for anyone who might be interested in helping me revive the abandoned project. That plugin at the moment is still the top result on Google when I search for NeoVim markdown viewer.

**Comment by jannis-baum** [comment]
First action item so far: Mention "Markdown" more in the description of the Vim plugin (people seem to look more for Vim plugins than standalone viewers because they don't come up with the idea of a standalone viewer)
